internal/proxy/proxy.go

```diff
@@ -88,6 +96,25 @@ type Config struct {
 	Dialer cloudsql.Dialer
 }
 
+// DialOptions interprets appropriate dial options for a particular instance
+// configuration
+func (c *Config) DialOptions(i *InstanceConnConfig) []cloudsqlconn.DialOption {
+	var opts []cloudsqlconn.DialOption
+
+	if i.IAMAuthN != nil {
+		opts = append(opts, cloudsqlconn.WithDialIAMAuthN(*i.IAMAuthN))
+	}
+
+	if i.PrivateIP != nil && *i.PrivateIP ||
+		i.PrivateIP == nil && c.PrivateIP {
```

Comment:
nit: seems like a line break is unnecessary here

Reply:
removed.